### DISCUSSION TITLE
pls help change autoload.lua to sort by date?

### DISCUSSION THREAD

**Discussion by ydqsbfsyi** [smplayer-dev/smplayer#940] [Q&A] [answered]
https://github.com/mpv-player/mpv/blob/master/TOOLS/lua/autoload.lua

This lua script can automatically add the current video file's folder as a playlist, it works great. Just I want the files sorted by date instead of filename, and I want the sort in descending order.

Never learned lua before, so I used AI and I got several versions of code, some got me wrong files count - it's accumulating when I click next button again and again; some failed to show playlist.

Could somebody please give me a hand?

**Comment by smplayer-dev** [comment, marked as answer]
The script won't work with smplayer. smplayer and mpv use different playlists.

**Reply by ydqsbfsyi** [reply]
Thanks for your help.
